bin/ci

```diff
@@ -8,44 +8,38 @@ require "optparse"
 @started_at = Time.now
 
 def main(options)
-  boot_images = get_additional_boot_images(options[:test_cases])
-  hetzner_server_st = Prog::Test::HetznerServer.assemble(vm_host_id: options[:vm_host_id], additional_boot_images: boot_images)
+  all_test_cases = YAML.load_file("config/e2e_test_cases.yml").to_h { [_1["name"], _1] }
+  boot_images = all_test_cases.values_at(*options[:test_cases]).flat_map { _1["images"] }
```

Comment:
jw: should we dedup here? does repeating boot images cause any issues/inefficiency? For eg. the postgres test I am adding will use the same image as the existing one

Reply:
Makes sense. I added `.uniq`. When we try to download the existing boot image to the host, it fails.